src/win/winsock.c

```diff
@@ -63,18 +67,17 @@ static BOOL uv__get_extension_function(SOCKET socket, GUID guid,
 
 
 BOOL uv__get_acceptex_function(SOCKET socket, LPFN_ACCEPTEX* target) {
-  const GUID wsaid_acceptex = WSAID_ACCEPTEX;
-  return uv__get_extension_function(socket, wsaid_acceptex, (void**)target);
+  *target = uv_wsa_acceptex;
```

Comment:
Rather than do this, how about we expose `uv_wsa_acceptex` and `uv_wsa_connectex` for internal use, so the code can call them directly?

Reply:
If `tcp.c` currently handles it in a graceful way, I think we'd better keep as it is? Otherwise hard crashes will be introduced.

Reply:
Yes, but when exactly would those be missing? We might no longer actually run into the problem.

Reply:
Yes, if you consider it a fatal error, I'm happy to do that.

Reply:
IMHO it is.